skimage/feature/tests/test_corner.py

```diff
@@ -467,7 +487,7 @@ def test_corner_orientations_astronaut():
                           2.47305654e+00, -1.63869275e+00,  5.46905279e-02,
                          -4.40598471e-01,  3.14918803e-01, -1.76069982e+00,
                           3.05330950e+00,  2.39291733e+00, -1.22091334e-01,
-                         -3.09279990e-01,  1.45931342e+00])
+                         -3.09279990e-01,  1.45931342e+00])[::-1]
```

Comment:
I don't understand why orders are reversed... I get that the order has changed, but why has it changed to the exact opposite?